src/include/cache.i

```diff
@@ -133,93 +133,6 @@ __wt_cache_status(WT_SESSION_IMPL *session, int *evictp, int *dirtyp)
 }
 
 /*
- * __wt_eviction_check --
- *	Wake the eviction server if necessary.
- */
-static inline int
-__wt_eviction_check(WT_SESSION_IMPL *session, int *fullp, int wake)
```

Comment:
No advantage, of course, but threads calling this function have already self-selected as not being terribly busy, right?

They're either about to stall (page-in), have no positioned cursors (cursor-enter), allocating a snapshot (txn-begin), or doing the "transaction idle check". What's the performance path that makes avoiding a function call interesting?

Regardless, happy to inline the front end of `__wt_cache_full_check`, just let me know.